cmd/influxd/run/server.go

```diff
@@ -144,7 +143,6 @@ func (s *Server) Close() error {
 type Service interface {
 	Open() error
 	Close() error
-	Addr() net.Addr
```

Comment:
Why this change? I don't object to it, but we just discussed it this morning.

I do think that `Addr` may not be the right call to have on `Service` so it's OK with me.